CHANGELOG

```diff
@@ -1,5 +1,9 @@
 v3.8.0 (XXXX-XX-XX)
```

Comment:
The feature will get only into 3.8.1, so the CHANGELOG needs adjustment before merging.